azure-functions-shared/Cargo.toml

```diff
@@ -10,7 +10,7 @@ edition = "2018"
 
 [dependencies]
 azure-functions-shared-codegen = { version = "0.11.0", path = "../azure-functions-shared-codegen" }
-tonic = "0.1.0-alpha.5"
+tonic = { git = "https://github.com/hyperium/tonic.git", branch = "master" }
```

Comment:
We won't be able to take this PR until `tonic` publishes a needed version.  We've run into problems in the past of doing lots of work on top of git-referenced crates and then being at their mercy to release before we can publish new crates.

Reply:
Understand completely. I don't think a new version is too far off

Reply:
It looks like Tonic `0.1.0-beta.1` was released recently with a dependency of a non-preview 0.3.0 futures.  We have a update PR for Tonic that's failing to build, so I'm going to try to bump it to fix that PR.